evutil.c

```diff
@@ -2379,7 +2379,7 @@ evutil_parse_sockaddr_port(const char *ip_as_string, struct sockaddr *out, int *
 		if ((int)sizeof(sin6) > *outlen)
 			return -1;
 		sin6.sin6_scope_id = if_index;
-		memset(out, 0, *outlen);
+		memset(out, 0, sizeof(sin6));
```

Comment:
Than memset could be simply removed, since below there is memcpy.

Reply:
Should we just remove this lines?

Reply:
OK, let's do this. Please submit another PR.

Reply:
Thanks. And make a pr #1585